src/autocomplete/databases/clickhouse/grammar/ClickHouseLexer.g4

```diff
@@ -8,342 +8,394 @@ lexer grammar ClickHouseLexer;
-SETTING                       : S E T T I N G;
-OPTION                        : O P T I O N;
-NONE                          : N O N E;
-AZURE                         : A Z U R E;
-HIVE                          : H I V E;
-MONGO                         : M O N G O;
-POSTGRES                      : P O S T G R E S;
-REDIS                         : R E D I S;
-SQLITE                        : S Q L I T E;
-ADDRESSTOLINEWITHINLINES      : A D D R E S S T O L I N E W I T H I N L I N E S;
-SQL                           : S Q L;
-SECURITY                      : S E C U R I T Y;
-DISPLAYSECRETSINSHOWANDSELECT : D I S P L A Y S E C R E T S I N S H O W A N D S E L E C T;
-POLICIES                      : P O L I C I E S;
-PROFILES                      : P R O F I L E S;
-ALLOW                         : A L L O W;
-GRANTS                        : G R A N T S;
-EXCEPT                        : E X C E P T;
-REVOKE                        : R E V O K E;
-
-FORMAT_TAB_SEPARATED                                      : 'TabSeparated';
-FORMAT_TAB_SEPARATED_RAW                                  : 'TabSeparatedRaw';
-FORMAT_TAB_SEPARATED_WITH_NAMES                           : 'TabSeparatedWithNames';
-FORMAT_TAB_SEPARATED_WITH_NAMES_AND_TYPES                 : 'TabSeparatedWithNamesAndTypes';
-FORMAT_TAB_SEPARATED_RAW_WITH_NAMES                       : 'TabSeparatedRawWithNames';
-FORMAT_TAB_SEPARATED_RAW_WITH_NAMES_AND_TYPES             : 'TabSeparatedRawWithNamesAndTypes';
-FORMAT_TEMPLATE                                           : 'Template';
-FORMAT_TEMPLATE_IGNORE_SPACES                             : 'TemplateIgnoreSpaces';
-FORMAT_CSV                                                : 'CSV';
-FORMAT_CSV_WITH_NAMES                                     : 'CSVWithNames';
-FORMAT_CSV_WITH_NAMES_AND_TYPES                           : 'CSVWithNamesAndTypes';
-FORMAT_CUSTOM_SEPARATED                                   : 'CustomSeparated';
-FORMAT_CUSTOM_SEPARATED_WITH_NAMES                        : 'CustomSeparatedWithNames';
-FORMAT_CUSTOM_SEPARATED_WITH_NAMES_AND_TYPES              : 'CustomSeparatedWithNamesAndTypes';
-FORMAT_VALUES                                             : 'Values';
-FORMAT_JSON                                               : 'JSON';
-FORMAT_JSON_AS_STRING                                     : 'JSONAsString';
-FORMAT_JSON_AS_OBJECT                                     : 'JSONAsObject';
-FORMAT_JSON_STRINGS                                       : 'JSONStrings';
-FORMAT_JSON_COLUMNS                                       : 'JSONColumns';
-FORMAT_JSON_COLUMNS_WITH_METADATA                         : 'JSONColumnsWithMetadata';
-FORMAT_JSON_COMPACT                                       : 'JSONCompact';
-FORMAT_JSON_COMPACT_COLUMNS                               : 'JSONCompactColumns';
-FORMAT_JSON_EACH_ROW                                      : 'JSONEachRow';
-FORMAT_JSON_STRINGS_EACH_ROW                              : 'JSONStringsEachRow';
```

Comment:
I had to change declaration of this tokens because actually they are case insensitive